diff --git a/chromium/scripts/build_ffmpeg.py b/chromium/scripts/build_ffmpeg.py
index 5b478be..14c6f99 100755
--- a/chromium/scripts/build_ffmpeg.py
+++ b/chromium/scripts/build_ffmpeg.py
@@ -300,6 +300,7 @@ def main(argv):
       '--enable-fft',
       '--enable-rdft',
       '--enable-static',
+      '--enable-nonfree',

       # Disable features.
       '--disable-bzlib',
@@ -551,9 +552,9 @@ def main(argv):

   # Google Chrome & ChromeOS specific configuration.
   configure_flags['Chrome'].extend([
-      '--enable-decoder=aac,h264,mp3',
-      '--enable-demuxer=aac,mp3,mov',
-      '--enable-parser=aac,h264,mpegaudio',
+    '--enable-decoder=aac,ac3,eac3,h264,mp1,mp2,mp3,mpeg4,mpegvideo,hevc,flv,dca,flac',
+    '--enable-demuxer=aac,h264,mp3,mp4,m4v,mpegvideo,mpegts,mov,avi,flv,dts,dtshd,vc1,flac',
+    '--enable-parser=aac,ac3,aac3,h261,h263,h264,mepgvideo,mpeg4video,mpegaudio,dca,hevc,vc1,flac',
   ])

   # ChromiumOS specific configuration.
